### ISSUE TITLE
HTTP client certificate auth is only available via Safari on iOS

### ISSUE THREAD

**Issue by cloventt** [mozilla/platform-tilt#30]
iOS Safari allows for HTTP client certificate auth if the certificate is loaded as a profile in iOS settings. Non-Safari browsers don’t appear to be allowed to access these profiles for authentication.

**Comment by estan** [comment]
Yes, I think this is due to iOS settings adding the installed certificates into the iOS Keychain under the Apple access group, meaning only Apple apps like Mail or Safari can access them from there. This forces non-Apple apps into implementing their own certificate management from scratch instead of leveraging the one offered by the OS, thus putting Firefox at a disadvantage compared to Apple's own Safari browser.

Apple is owning up to this malpractice at https://developer.apple.com/library/archive/qa/qa1745/_index.html, but offers no rationale for it.
